breizhcrops/datasets/breizhcrops.py

```diff
@@ -221,22 +220,47 @@ def load_classmapping(self, classmapping):
         if self.verbose:
             print(f"read {self.nclasses} classes from {classmapping}")
 
-    def load(self, csv_file):
+    def load_raw(self, csv_file):
         """['B1', 'B10', 'B11', 'B12', 'B2', 'B3', 'B4', 'B5', 'B6', 'B7', 'B8',
-       'B8A', 'B9', 'QA10', 'QA20', 'QA60', 'doa', 'label', 'id']"""
+               'B8A', 'B9', 'QA10', 'QA20', 'QA60', 'doa', 'label', 'id']"""
 
         sample = pd.read_csv(os.path.join(self.csvfolder, os.path.basename(csv_file)), index_col=0).dropna()
         # convert datetime to int
         sample["doa"] = pd.to_datetime(sample["doa"]).astype(int)
         sample = sample.groupby(by="doa").first().reset_index()
-        X = np.array(sample[self.bands].values)
+
+        return sample
+
+    def load(self, csv_file):
+        sample = self.load_raw(csv_file)
+        X = np.array(sample[self.bands[1:17]].values)
```

Comment:
this `self.bands[1:17]` is not clear. It would be nicer to manually define a list of bands
```
selected_bands = ["B01","B02", ...]
``` 
and then select the sample dataframe by these selected bands 
```
sample[selected_bands]
```
Also. 1:17 may be not correct? It appears that it skips the first band in self.bands?